csharp/src/Drivers/BigQuery/BigQueryConnection.cs

```diff
@@ -136,30 +136,30 @@ internal BigQueryClient Open(string? projectId = null)
                     if (projectId.Equals(BigQueryConstants.PublicProjectId, StringComparison.OrdinalIgnoreCase))
                     {
                         projectId = BigQueryConstants.DetectProjectId;
-                        activity?.AddBigQueryTag("change_public_projectId_to_detect_project_id", projectId);
+                        activity?.AddBigQueryTag("change_public_projectId_to_detect_project_id", projectId, isPii: false);
                     }
                 }
 
                 // the billing project can be null if it's not specified
                 if (this.properties.TryGetValue(BigQueryParameters.BillingProjectId, out billingProjectId))
                 {
-                    activity?.AddBigQueryParameterTag((BigQueryParameters.BillingProjectId), billingProjectId);
+                    activity?.AddBigQueryParameterTag((BigQueryParameters.BillingProjectId), billingProjectId, isPii: false);
```

Comment:
I could see this potentially being pii, baesd on how people name their projects.

Reply:
Done